common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/service/SaltService.java

```diff
@@ -0,0 +1,29 @@
+package app.coronawarn.datadonation.common.persistence.service;
+
+import app.coronawarn.datadonation.common.persistence.repository.ppac.android.SaltRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SaltService {
```

Comment:
can we add here `@Profile("e2e-test")`???

then we might not need to touch prod-code